src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -183,8 +192,8 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
   // all students of the course
   studentsOfCourse: Student[] = [];
   emailOfStudentToPreview: string = '';
-  // instructors which can be previewed as
-  instructorsCanBePreviewedAs: Instructor[] = [];
+  // all instructors of course
```

Comment:
Sorry rly minor nit, but if you are going to make other changes do you mind changing this as well. The comment doesn't match the students one :). If not convenient just leave it haha.

```suggestion
  // all instructors of the course
```